plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -657,4 +663,9 @@ public static boolean isFileBasedConflictDetectionEnabled(ConnectorSession sessi
     {
         return session.getProperty(FILE_BASED_CONFLICT_DETECTION_ENABLED, Boolean.class);
     }
+
+    public static int getIcebergMaxPartitionsPerWriter(ConnectorSession session)
```

Comment:
No need to include "get" in the session property name, just `maxPartitionsPerWriter(ConnectorSession session)` will do